### ISSUE TITLE
Transcoding .mkv to .mp4: File growth

### ISSUE THREAD

**Issue by stefan-reich** [jcodec/jcodec#167]
I'm transcoding a 13.4 MB .mkv, and it's still running, the output file is now at 219 MB.

All I'm doing is `TranscodeMain.main(new String[] {inputFile, outputFile});`

**Comment by svitvitskiy** [comment]
Use video copy, I bet MKV is also h.264 so you don't need to transcode only
transmux.

JCodec might be able to transcode but it's extremely bad at it both in
terms of time and bitrate.

**Comment by stefan-reich** [comment]
"Video copy" = ffmpeg or something?

**Comment by svitvitskiy** [comment]
no, I meant in transcoder command line you can specify '--o:vc=copy'
The same way in Transcode interface there's a boolean flag 'videoCopy'.

If you set that one the source frames will not be transcoded but rather 'copied over' to the new file. If you just want to move your video from say MKV to MP4 you don't have to transcode it, you can just take encoded video frame and store them in a different file. That's what videoCopy argument stands for.

An equivalent ffmpeg command flag is '-vc=copy'.
